android/source/VideoLocker/src/org/edx/mobile/base/MainApplication.java

```diff
@@ -40,18 +37,22 @@ private void init() {
         // initialize logger
         Logger.init(this.getApplicationContext());
 
+        // setup environment
         Environment env = new Environment();
         env.setupEnvironment(this.getApplicationContext());
 
-        RequestManager.init(this);
+        // setup image cache
         createImageCache();
 
         // initialize SegmentIO, empty writeKey is handled in SegmentTracker
         SegmentFactory.makeInstance(this);
 
+        // initialize Fabric
         if(Config.getInstance().getFabricKey() != null) {
             Fabric.with(this, new Crashlytics());
         }
+
+        // initialize NewRelic with crashlytics disabled
```

Comment:
This should say "crash reporting" not "Crashlytics", since that's a brand name that's part of fabric.